app/forms.py

```diff
@@ -21,4 +21,3 @@ class ResetPasswordForm(FlaskForm):
     username = StringField('Enter Your Name', validators=[InputRequired()])
     password = PasswordField('Password', validators=[InputRequired()])
     confirm_password = PasswordField('Confirm New Password', validators=[InputRequired()])
-# End of forms.py
```

Comment:
No new line at end of file.

Reply:
There is a line there, I just removed the comment that I felt unnecessary.